### ISSUE TITLE
The results reproduced are quite different from those in the article

### ISSUE THREAD

**Issue by 30410B** [Hk416mod2/UniVesselSeg#2]
This is an excellent work， but when i reproduce your work， I got a very bad result，could you please provide your hyper parameter settings during traing stage of CHSEDB1 dataset.

**Comment by Hk416mod2** [comment]
Thank you very much for your kind words and for reproducing our work!

I apologize for the confusion. The model in our work was trained on a _combined dataset_ that included CHASE_DB1 along with other vessel segmentation datasets. We didn’t train it on any single dataset such as CHASE_DB1, which contains very limited samples. From our tests, training solely on such small dataset tends to produce unstable or poor results, possibly due to the limited data size, the use of Batch Normalization layers, or other training settings.

For better results, you can directly use the pretrained weights we provided which were trained on the combined datasets and it should have reasonable performance on CHASE_DB1.

Thank you again for your feedback and understanding! Please feel free to reach out if you have any further questions.

**Comment by Xiaohuan-Ding** [comment]
I reproduce the test results of OCTA modality datasets with the pretrained weights. But I find that the test results (especially OCTA500-3M and OCTA500-6M ) is lower than those reported in paper. Could you please state the specific layers used in OCTA500 dataset? I used the OCTA_ILM_OPL and GT_LargeVessel for testing, and I don't know if it's consistent with your experiments.

**Comment by Hk416mod2** [comment]
Thank you very much for your interest in reproducing our method. For the OCTA500 dataset, we indeed used OCTA_ILM_OPL as the images and GT_LargeVessel as the GT. Here we would like to clarify one preprocessing detail that may account for your performance discrepancy: during early data preprocessing, we found the orientation of OCTA500 dataset differs from other OCTA datasets (e.g., ROSSA), and therefore we applied rotation to the OCTA500 dataset (imgs & gts, rotation only) to ensure directional consistency. Here's an easy example and our processed OCTA500 dataset.

<img width="1227" height="853" alt="Image" src="https://github.com/user-attachments/assets/6bee937b-488f-4278-a14d-1f4ef582385a" />

Processed OCTA500 dataset: https://drive.google.com/file/d/1nHP-PQkVevhn6N8axnTlxn1yeEAqr3E2/view?usp=drive_link

The pretrained model should be able to achieve similar performance to that in the paper on this processed dataset.

**Comment by Xiaohuan-Ding** [comment]
Thank you so much for your preprocessed OCTA-500 dataset.  I can achieve the same performance as that reported in the paper. 

I still have some questions about the datasets. 

1. In Dataset/dataset_split.json, the number of STARE dataset split is 16:2:2, which is different from the Table I in the paper. 

2. The original download link for ORVS dataset is unavailable. Could you please share a new download link for ORVS dataset?

**Comment by Hk416mod2** [comment]
For STARE dataset, please follow the setting in Dataset/dataset_split.json.

For ORVS dataset, it appears that their official website(https://github.com/AbdullahSarhan/ICPRVessels) is unavailable??? Here's the ORVS dataset we downloaded before: https://drive.google.com/file/d/1EmIxkRXBPK8LaQj2nsrPNyOTxRNpDyuH/view?usp=drive_link

Hope this will help you :)
